querybook/webapp/lib/sql-helper/sql-lexer.ts

```diff
@@ -807,6 +813,7 @@ export function findTableReferenceAndAlias(statements: IToken[][]) {
     return {
         references,
         aliases,
+        activeSchema,
```

Comment:
looks like `activeSchema` is just the same as `defaultSchema`, you dont need to define a new variable, but just return the defaultSchema.